integration_tests/src/tests/commit_post_test.rs

```diff
@@ -64,7 +62,7 @@ fn setup(v: &dyn VM) -> (MinerInfo, SectorInfo) {
 
     // precommit and advance to prove commit time
     let sector_number: SectorNumber = 100;
-    let infos = precommit_sectors_v2(
+    let _ = precommit_sectors_v2(
```

Comment:
Rename this to v3 too?

Reply:
Hmm but the name of the latest  pre_commit_sector_batch2? I agree the 2 vs 3 thing is confusing. We should probably FIP removal of pre_commit_sector_batch and then we can just gives these functions back their old names.

Reply:
Oops, sorry. Leave as v2.